### DISCUSSION TITLE
Change Header Titles-question

### DISCUSSION THREAD

**Discussion by jose-franco-PT** [joaomlourenco/novathesis#241] [Q&A] [answered]
Good evening, first of all I would like to thank you for sharing this project with the entire community.
Right now I'm writing my master's thesis, with the most up-to-date template 6.5.3 in Overleaf Platform. After doing some research  I can't find a solution to the following problem:

-On the pages relating to chapters and subchapters, the titles of the chapters/subchapters always appear in the headers, as shown in the figure below:

![imagem](https://user-images.githubusercontent.com/83010884/140835077-1d9a7908-235b-48fc-8f51-dc9735916870.png)

For example, in the case of subchapters it appears like this:
![imagem](https://user-images.githubusercontent.com/83010884/140835186-f082bacb-40cf-4208-ab4a-3d2d56a7fae7.png)

-Taking into account the last figures, my question is to understand if it is possible to change in all headings, the titles of the chapters, for the main title of the thesis.

Thank you for your attention,

Best Regards,

José Franco

**Comment by joaomlourenco** [comment]
The good thing with LaTeX is that you can always do whatever you want! 😃
The bad thing is that sometimes it is not that easy.

What is your real problem?  Is it the fact that the title in the subsection spans over two lines?  If that is your real problem, you can solve it easily by declaring an abbreviated header for your section as:
```latex
\section[abbreviated version of your header]{Cenário de Motivação — Internet das Coisas e Sistemas Ciberfísicos}
```
e.g.,
```latex
\section[Cenário de Motivação — IoT e Sistemas Ciberfísicos]{Cenário de Motivação — Internet das Coisas e Sistemas Ciberfísicos}
```
The longer version (in "{…}") will be used for the section title.
The shorter version (in "[…]") will be used for the table of contents and the page header.

**Reply by b-pereira** [reply]
>  if it is possible to change in all headings, the titles of the chapters, for the main title of the thesis.

@joaomlourenco I think that @jose-franco-PT wants the _DISSERTATION'S TITLE **FIXED** (as in pinned)_ in the headers of **all the pages**.

That is, instead of "Chapter 1. Introduction" to "Internet das Coisas e Sistemas Ciberfísicos", assuming that "Internet das Coisas e Sistemas Ciberfísicos" is the dissertation's title.

**Comment by jose-franco-PT** [comment, marked as answer]
Good morning, first of all, I apologize for not being very clear actually.
Honestly, what I need, if possible, is to show the title of the thesis pinned in the header of all pages, as @b-pereira said.

I've already been researching the project and in fact I'm not able to understand how to make this change.

Once again, thank you for your attention.

Best Regards,
José Franco
